### PR TITLE
feat: Proposing new features for AI. Using myself LLM

### PR DESCRIPTION
**Pull request by mrkito**
closes: https://github.com/keephq/keep/issues/2704

### CODE REVIEW

File: docs/deployment/configuration.mdx [hunk 1]
```diff
@@ -135,6 +135,7 @@ OpenAPI configuration is used for integrating with OpenAI services. These settin
 | Env var | Purpose | Required | Default Value | Valid options |
 |:-------------------:|:-------:|:----------:|:-------------:|:-------------:|
 | **OPENAI_API_KEY** | API key for OpenAI services | No | None | Valid OpenAI API key |
+| **OPENAI_BASE_URL** | URL for OpenAI API. It's need for use yourself LOCAL LLM instead OpenAi LLM. | No | None | Valid OpenAI API url |
```
Comment (Matvey-Kuk) [inline docs/deployment/configuration.mdx:138]: Please provide examples here, I assume default value is not None.

**Review by Matvey-Kuk — CHANGES_REQUESTED** [review]

File: docs/deployment/configuration.mdx (hunk 1 above)
Comment (Matvey-Kuk) [reply docs/deployment/configuration.mdx:138]: Found in the lib code, it's `https://api.openai.com/v1`

Comment (talboren) [reply docs/deployment/configuration.mdx:138]: @Matvey-Kuk can this be merged in?

Comment (Matvey-Kuk) [inline docs/deployment/configuration.mdx:138]:
```suggestion
| **OPENAI_BASE_URL** | URL for OpenAI API. It's need for use yourself LOCAL LLM instead OpenAi LLM. | No | https://api.openai.com/v1 | Valid OpenAI API url |
```

**Review by Matvey-Kuk — APPROVED** [review]

**Comment by talboren** [comment]
@mrkito can we have you sign the CLA so we can merge this one? :)

**Comment by mrkito** [comment]
> @mrkito can we have you sign the CLA so we can merge this one? :)

Yes, i signed

**Comment by talboren** [comment]
> > @mrkito can we have you sign the CLA so we can merge this one? :)
> 
> Yes, i signed

Still looks invalid. Can you double check? 🙏🏼

**Comment by Matvey-Kuk** [comment]
I'm closing this one as stale. Please open the new one once you sign the CLA, thank you!

**Pull request closed without merging**